packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -20,19 +20,15 @@ import {
 import { IDS } from '../../common/constants';
 import { isMobile, verifyParamExists } from '../../common/testAppUtils';
 import memoizeOne from 'memoize-one';
-import { FontIcon, Icon, IContextualMenuItem, mergeStyles } from '@fluentui/react';
+import { Icon, IContextualMenuItem, mergeStyles } from '@fluentui/react';
 import { fromFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { MockCallAdapter } from './mocks/MockCallAdapter';
+import { TestCallingState } from '../TestCallingState';
 
 const urlSearchParams = new URLSearchParams(window.location.search);
 const params = Object.fromEntries(urlSearchParams.entries());
 
-// Required params
-const displayName = verifyParamExists(params.displayName, 'displayName');
-const token = verifyParamExists(params.token, 'token');
-const groupId = verifyParamExists(params.groupId, 'groupId');
-const userId = verifyParamExists(params.userId, 'userId');
-
-// Optional params
+const state = JSON.parse(params.state) as TestCallingState;
```

Comment:
The name `state` is too generic. How about `mockCallingState`. Also see my comment below about not using `useMockAdapter` at all.